<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThirdDialog</class>
 <widget class="QDialog" name="ThirdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>  G-mail :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>133</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>                      0123...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gmail">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="placeholderText">
    <string>  joey @gmail.com</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background:rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background:rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background:rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>Add Resort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
background:rgb(0, 0, 255)</string>
   </property>
   <property name="text">
    <string>Show Resort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>30</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(0, 0, 127)</string>
   </property>
   <property name="text">
    <string>customer care</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>101</y>
     <width>63</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;MS Shell Dlg 2&quot;;
background: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>User Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>       joey  islam</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
